core/templates/pages/exploration-player-page/learner-experience/input-response-pair.component.spec.ts

```diff
@@ -238,4 +249,15 @@ describe('InputResponsePairComponent', () => {
       ExplorationPlayerConstants.AUDIO_HIGHLIGHT_CSS_CLASS
     );
   });
+
+  it('should return empty css class for feedback audio highlight', () => {
```

Comment:
we should also check if it returns non-empty css class for other cases

Reply:
I added more unit tests.